modyn/supervisor/internal/grpc_handler.py

```diff
@@ -558,22 +573,17 @@ def _prepare_evaluation_request(
             )
 
         start_evaluation_kwargs = {
-            "trained_model_id": trained_model_id,
+            "model_id": model_id,
             "dataset_info": DatasetInfo(dataset_id=dataset_id, num_dataloaders=dataloader_workers),
             "device": device,
-            "amp": amp,
             "batch_size": batch_size,
             "metrics": metrics,
-            "model_id": model_id,
-            "model_configuration": EvaluatorJsonString(value=model_config),
             "transform_list": transform_list,
             "bytes_parser": EvaluatorPythonString(value=bytes_parser_function),
             "label_transformer": EvaluatorPythonString(value=label_transformer),
         }
 
-        cleaned_kwargs = {k: v for k, v in start_evaluation_kwargs.items() if v is not None}
```

Comment:
Are you 100% sure we do not need the cleaned kwargs anymore?

Reply:
Yes. However, I can revert the change if you want.